mod/groups/start.php

```diff
@@ -496,34 +496,45 @@ function groups_create_event_listener($event, $object_type, $object) {
  * Return the write access for the current group if the user has write access to it.
  */
 function groups_write_acl_plugin_hook($hook, $entity_type, $returnvalue, $params) {
-	$page_owner = elgg_get_page_owner_entity();
-	$user_guid = $params['user_id'];
+	
+	if (empty($params) || !is_array($params)) {
+		return $returnvalue;
+	}
+	
+	$user_guid = sanitise_int(elgg_extract('user_id', $params), false);
 	$user = get_user($user_guid);
-	if (!$user) {
+	if (empty($user)) {
 		return $returnvalue;
 	}
```

Comment:
Because we're not unanimous about the stuff above. This prevents merging the stuff below.